views/layout.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<!--[if IE 9]>
+<html class="lt-ie10" lang="en">
+<![endif]-->
+<html class="no-js" lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Oh Chit!</title>
+  <!-- normalize.css and modernizr.js help with browser compatibility -->
+  <link rel="stylesheet" href="/css/normalize.css">
+  <link rel="stylesheet" href="/css/foundation.css">
+  <script src="js/vendor/modernizr.js"></script>
+</head>
+<body>
+<div class="row">
+  <div class="small-3 small-centered columns"></div>
+  <div class="small-6 small-centered columns">
+    <nav class="top-bar" data-topbar role="navigation">
+      <ul class="title-area">
+        <li class="name">
+          <h1><a href="/"><strong>Oh Chit!</strong></a></h1>
+        </li>
+      </ul>
+    </nav>
+
+    <%= yield %></div>
```

Comment:
Is this the right place for the `yield` block? You need to format the html. The `</div> tag seems off...